compiler/ballerina-lang/src/main/resources/compiler.properties

```
@@ -1138,6 +1135,12 @@ error.cannot.define.constant.with.type=\
 error.expression.is.not.a.constant.expression=\
   expression is not a constant expression
 
+error.invalid.const.expression=\
+  invalid constant expression, reason ''{0}''
```

Minor change, how about something like
```suggestion
  invalid constant expression: {0}
```
instead?